controller/webapp/views/rap_views.py

```diff
@@ -177,11 +177,20 @@ def cancel(request, *, token_backends, request_obj: CancelRequest):
             status=404,
         )
 
-    log.info(
-        "Cancelling actions for job_request %s: %s",
-        request_obj.rap_id,
-        request_obj.actions,
-    )
+    if len(actions_to_cancel) > 1:
```

Comment:
I know they should equate to the same thing, but it feels a bit odd to look at `len(actions_to_cancel)` here but then log `len(request_obj.actions)`.